### DISCUSSION TITLE
.MachineID seems to be Null

### DISCUSSION THREAD

**Discussion by enyachoke** [kairos-io/kairos#2987] [Q&A]
Hello folks. I am having an issue with the machine id not being set when I use node-{{ trunc 4 .MachineID }} The node name ends up being just `node` This used to work fine until I reinstalled my cluster recently.

**Comment by jimmykarily** [comment]
Is it happening all the time? Can you specify the version of Kairos you are using, the config and the steps you are taking?
We need a way to reproduce it. I'll create a ticket out of this discussion anyway to check if this is still working in general.

**Reply by jimmykarily** [reply]
https://github.com/kairos-io/kairos/issues/3038
